### PR TITLE
Converting  the form fields for VT Entity Method Task to LDS

### PR DESCRIPTION
**Pull request by rdsande**
Converting the fields, removing the old table tags and substituting them with the div and classes from the LDS system

### CODE REVIEW

File: Smarty/templates/com_vtiger_workflow/EditTask.tpl [hunk 1]
```diff
@@ -28,84 +28,191 @@
...
-
-		<table class="tableHeading" border="0"  width="100%" cellspacing="0" cellpadding="5">
-			<tr>
-				<td class="big" nowrap="nowrap">
-					<strong>{$MOD.LBL_TASK_OPERATIONS}</strong>
-				</td>
-			</tr>
-		</table>
+		</div>
+
```
Comment (joebordes) [inline Smarty/templates/com_vtiger_workflow/EditTask.tpl:214]: please eliminate one of these two empty lines

File: Smarty/templates/com_vtiger_workflow/taskforms/VTEntityMethodTask.tpl [hunk 2]
```diff
@@ -8,17 +8,30 @@
  * All Rights Reserved.
  ************************************************************************************/
 -->*}
+
```
Comment (joebordes) [inline Smarty/templates/com_vtiger_workflow/taskforms/VTEntityMethodTask.tpl:11]: in the future do not add empty lines like this

File: Smarty/templates/com_vtiger_workflow/EditTask.tpl (hunk 1 above)
Comment (rdsande) [reply Smarty/templates/com_vtiger_workflow/EditTask.tpl:214]: Ok

File: Smarty/templates/com_vtiger_workflow/taskforms/VTEntityMethodTask.tpl (hunk 2 above)
Comment (rdsande) [reply Smarty/templates/com_vtiger_workflow/taskforms/VTEntityMethodTask.tpl:11]: Alright

**Pull request closed without merging**